azurerm/internal/services/devtestlabs/tests/resource_arm_dev_test_global_shutdown_schedule_test.go

```diff
@@ -0,0 +1,243 @@
+  resource_group_name  = "${azurerm_resource_group.test.name}"
+  virtual_network_name = "${azurerm_virtual_network.test.name}"
+  address_prefix       = "10.0.2.0/24"
+}
+
+resource "azurerm_network_interface" "test" {
+  name                = "acctnic-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  ip_configuration {
+    name                          = "testconfiguration1"
+    subnet_id                     = "${azurerm_subnet.test.id}"
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+resource "azurerm_virtual_machine" "test" {
+  name                  = "acctvm-%d"
+  location              = "${azurerm_resource_group.test.location}"
+  resource_group_name   = "${azurerm_resource_group.test.name}"
+  network_interface_ids = ["${azurerm_network_interface.test.id}"]
+  vm_size               = "Standard_B2s"
+
+  storage_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "18.04-LTS"
+    version   = "latest"
+  }
+
+  storage_os_disk {
+    name              = "myosdisk-%d"
+    caching           = "ReadWrite"
+    create_option     = "FromImage"
+    managed_disk_type = "Standard_LRS"
+  }
+
+  os_profile {
+    computer_name  = "hostname"
+    admin_username = "testadmin"
+    admin_password = "Password1234!"
+  }
+
+  os_profile_linux_config {
+    disable_password_authentication = false
+  }
+}
+`, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger)
+}
+
+func testAccAzureRMDevTestLabGlobalShutdownSchedule_autoShutdownBasic(data acceptance.TestData) string {
+	template := testAccAzureRMDevTestLabGlobalShutdownSchedule_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_dev_test_global_shutdown_schedule" "test" {
+  location           = "${azurerm_resource_group.test.location}"
+  target_resource_id = "${azurerm_virtual_machine.test.id}"
+
+  daily_recurrence {
+    time = "0100"
+  }
+
+  time_zone_id = "Pacific Standard Time"
+
+  notification_settings {
+    status = "Disabled"
+  }
+
+  tags = {
+    environment = "Production"
+  }
+}
+`, template)
+}
+
+func testAccAzureRMDevTestLabGlobalShutdownSchedule_autoShutdownBasicUpdate(data acceptance.TestData) string {
+	template := testAccAzureRMDevTestLabGlobalShutdownSchedule_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_dev_test_global_shutdown_schedule" "test" {
+  location           = "${azurerm_resource_group.test.location}"
+  target_resource_id = "${azurerm_virtual_machine.test.id}"
+  status             = "Disabled"
+
+  daily_recurrence {
+    time = "1100"
+  }
+
+  time_zone_id = "Central Standard Time"
+
+  notification_settings {
+    time_in_minutes = 30
+    webhook_url     = "https://www.bing.com/2/4"
+    status          = "Enabled"
+  }
+
+  tags = {
+    environment = "Production"
```

Comment:
Make sure caps work for keys
```suggestion
    Environment = "Production"
```

Reply:
Done.